services/cd-service/pkg/repository/transformer.go

```diff
@@ -343,6 +343,11 @@ func (r *transformerRunner) Execute(t Transformer, transaction *sql.Tx) error {
 		r.Stack[idx-1] = append(r.Stack[idx-1], msg)
 	}
 	r.Stack = r.Stack[:idx]
+	if r.State.DBHandler.ShouldUseOtherTables() {
```

Comment:
There is already a mechanism to reload the overview, so these lines don't seem necessary. See `r.notify.Notify()` in repository.go